### ISSUE TITLE
0.8.12 causing key not found :step error

### ISSUE THREAD

**Issue by ehelms** [Dynflow/dynflow#192]
The latest release of Dynflow 0.8.12 is currently breaking unit tests on the Katello project. I have not run the application to verify if functionality is broken yet, however, I thought I would bring this breakage to light in case this minor update did break compatibility. You can see our test errors at [1].

```
KeyError: key not found: :step
    /var/lib/workspace/workspace/test_katello_core/database/postgresql/ruby/2.2/slave/fast/plugin/test/actions/katello/host/package_test.rb:73:in `block (2 levels) in <class:RemoveTest>'
    /var/lib/workspace/workspace/test_katello_core/database/postgresql/ruby/2.2/slave/fast/plugin/test/actions/katello/host/package_test.rb:96:in `block (3 levels) in <class:RemoveTest>' (KeyError)
/usr/local/rvm/gems/ruby-2.2.3@test_katello_core-3/gems/dynflow-0.8.12/lib/dynflow/action.rb:95 
```

[1] http://ci.theforeman.org/job/test_katello_core/19456/testReport/

**Comment by iNecas** [comment]
Thanks for heads up, this is cleanly a regression we will fix today in 0.8.13, stay tuned

**Comment by iNecas** [comment]
dynflow-0.8.13 with the fix was just released, should "make the Katello tests green again" :)
